samples/python/calibrate.py

```diff
@@ -140,7 +142,9 @@ def processImage(fn):
 
         if debug_dir:
             vis = cv.cvtColor(img, cv.COLOR_GRAY2BGR)
-            cv.drawChessboardCorners(vis, pattern_size, corners, found)
+            # cv.drawChessboardCorners(vis, pattern_size, corners, found)
+            cv.aruco.drawDetectedCornersCharuco(vis, corners, charucoIds=_charucoIds)
```

Comment:
sample supports both chessboards and charuco boards. The change breaks the first one.